hashes/Fedora-Cloud-Base-38-1.6.x86_64-20231207190935.qcow2.sha512

```diff
@@ -0,0 +1 @@
+875253d1b82e69b928f7ec52dda7da009c8b11033a0b86c2ba1cf141425afff93667117c4c0d64bd903ff35068d66913d0e4aa3b29213816ddbd15a616765253  Fedora-Cloud-Base-38-1.6.x86_64-20231207190935.qcow2
```

Comment:
Both sha512 files are missing the * before the filename. This is needed for some hash checks to succeed iirc.